src/App.js

```diff
@@ -1,25 +1,107 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState, useEffect } from "react"
+import Menu from "./components/Menu"
+import Question from "./components/Question"
+import { nanoid } from "nanoid"
+import "./App.css"
+const App = () => {
+  const [started, setStarted] = useState(false)
+  const [count, setCount] = useState(0)
+  const [correct, setCorrect] = useState(0)
+  const [checked, setChecked] = useState(false)
+  const [questions, setQuestions] = useState([])
+  const shuffleArray = (arr) => arr.sort(() => Math.random() - 0.5)
 
-function App() {
+  useEffect(() => {
+    async function getQuestion() {
+      const res = await fetch('https://opentdb.com/api.php?amount=5&category=18&encode=base64')
+      const data = await res.json()
+      let q = []
```

Comment:
q name is not descriptive, it is hard for someone who is reading your code to know what q is.